test/term.test.ts

```diff
@@ -32,100 +32,96 @@ describe('Terms', () => {
         const input = readFileSync(inputPath, 'utf8');
         const result = transformYfm(input, inputPath);
 
-        expect(clearRandomId(result)).toEqual(
-            '<h1>Web</h1>\n' +
-                '<p>The <i class="yfm yfm-term_title" term-key=":html" aria-describedby=":html_element" id="">HTML</i> specification</p>\n' +
-                '<template id=":html_template"><dfn class="yfm yfm-term_dfn" id=":html_element" role="tooltip"><p>The HyperText Markup Language or <strong>HTML</strong> is the standard markup language for documents designed to be displayed in a web browser.</p>\n' +
-                '</dfn></template>',
-        );
+        expect(clearRandomId(result)).toEqual(`\
```

Comment:
why do we have it in this order.

i mean why do we have this order: first content of the term definition and then term itself.

whats gonna happen if we use term inside paragraph?

term link is inline thing.

if we render content right before or after inline token, wouldn't it break everything?

e.g. we can use `term link` inside paragraph, and then we will try to render `term content` inside paragraph.

term content is container(at least it is promoted as such), will paragraph be able to render container block inside of it?